model-optimizer/extensions/ops/ONNXResize11.py

```diff
@@ -35,25 +35,28 @@ def onnx_resize_infer(node: Node):
         if input_shape is None:
             return
 
-        num_of_in_nodes = len(node.in_nodes())
-        assert num_of_in_nodes in {3, 4}, \
-            "Node {} with op {} number of inputs must be equal to 3 or 4.".format(node.name, node.op)
+        assert (node.is_in_port_connected(0) and (node.is_in_port_connected(2) or node.is_in_port_connected(3))), \
+            "Scales or sizes inputs must be connected to Node {} with op {}.".format(node.soft_get("name", node.id),
+                                                                                     node.soft_get("op", node.id))
 
         assert node.coordinate_transformation_mode != 'tf_crop_and_resize', \
-            'Mode tf_crop_and_resize is not supported for op {} with name {}'.format(node.op, node.name)
+            'Mode tf_crop_and_resize is not supported for op {} with name {}'.format(node.soft_get("op", node.id),
+                                                                                     node.soft_get("name", node.id))
 
-        if num_of_in_nodes == 3:
+        if not node.is_in_port_connected(3):
```

Comment:
`if node.is_in_port_connected(2)`

Reply:
I am not shure that it is good solution. According to onnx documentation input 2 can be connected with input 3 but tensor will be empty. But if input 3 connected not need check input 2.

Reply:
Do you have a real model where input 2 is connected with empty tensor? I think that in this case the input port 2 will not be connected at all. Please, double check that and post an update here.

Reply:
drn-d-54 from e2e test contain that case.